packages/module/sass/react-catalog-view-extension/_variables.scss

```diff
@@ -1,5 +1,5 @@
-$vertical-tab-pf-color: var(--pf-v5-global--Color--100) !default;
-$vertical-tab-pf-active-color: var(--pf-v5-global--active-color--100) !default;
+$vertical-tab-pf-color: var(--pf-v6-global--Color--100) !default;
+$vertical-tab-pf-active-color: var(--pf-v6-global--active-color--100) !default;
```

Comment:
I'm not sure where all these vars are used, but looks like this is probably the update we want to make? FWIW they're used to create the vars below in the `:root` block - `--vertical-tab-pf-color` and `--vertical-tab-pf-active-color`

```suggestion
$vertical-tab-pf-color: var(--pf-t--global--text--color--regular) !default;
$vertical-tab-pf-active-color: var(--pf-t--global--text--color--brand--clicked) !default;
```

It looks like you updated the use of `--vertical-tab-pf-active-color` in these 3 spots to just use the brand tokens directly:
* https://github.com/patternfly/react-catalog-view/blob/bafc9de25f809f04ca90285e22b1203099ae3bea/packages/module/sass/react-catalog-view-extension/_vertical-tabs.scss#L26
* https://github.com/patternfly/react-catalog-view/blob/bafc9de25f809f04ca90285e22b1203099ae3bea/packages/module/sass/react-catalog-view-extension/_vertical-tabs.scss#L44
* https://github.com/patternfly/react-catalog-view/blob/bafc9de25f809f04ca90285e22b1203099ae3bea/packages/module/sass/react-catalog-view-extension/_vertical-tabs.scss#L47

IMO that's fine, but you may also consider updating this line to use `--pf-t--global--text--color--regular`, which would be inline with the 3 updates above https://github.com/patternfly/react-catalog-view/blob/bafc9de25f809f04ca90285e22b1203099ae3bea/packages/module/sass/react-catalog-view-extension/_vertical-tabs.scss#L16

If there are no other uses of `--vertical-tab-pf-active-color` and `--vertical-tab-pf-color`, you may be able to nix this stylesheet. Just a thought! That may be a horrible idea, since I really have no idea how this project is setup 😅